exchanges/bitstamp.js

```diff
@@ -89,6 +89,9 @@ Trader.prototype.getFee = function(callback) {
 Trader.prototype.buy = function(amount, price, callback) {
   var args = _.toArray(arguments);
   var set = function(err, result) {
+    if (typeof result === 'undefined' || !result) {
```

Comment:
if this if check gets hit the result object with not have an ID property, it might not crash anymore but it still causes Gekko to not be able to monitor the order (since this will pass `undefined` to Gekko).

Reply:
I'm not actually sure where the result object has come from and can you clarify "gets hit the result object with not have an ID property"